src/main/kotlin/com/github/adeynack/jsoncmp/ComparisonBase.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.adeynack.jsoncmp
+
+import java.io.File
+
+abstract class ComparisonBase {
```

Comment:
Mmmm and now that we can insert code in interfaces, there is no reason (in this kind of case) not to. Right!